decaporg/decap-cms
# fix(markdown): convert inline CSS from Google Docs to Markdown

**Summary**

Extends the HTML to Markdown conversion to better support bold and italic formatting from Google Docs, which generates inline styles on a `span` element instead of strong/b/em/i type elements.

This makes it possible to copy limited formatting from Google Docs into the Markdown editor.

**Test plan**

1. Create a Google Docs document with bold, italic, bold & italic, and a bold & italic link.
2. Open the Posts collection in the test backend
3. Paste into the body and check the formatting matches

Example - copying formatted text from Google Docs into the Markdown editor and it's displaying correctly on the preview:

![image](https://github.com/user-attachments/assets/e2d00b8f-66ea-4518-ae7c-4bf9b23fca6f)

**Checklist**

Please add a `x` inside each checkbox:
